cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -118,6 +118,9 @@ spec:
             - name: tcp-model
               containerPort: 9003
               protocol: TCP
+             {{- with .Values.kubecostModel.extraPorts }}
+             {{- toYaml . | nindent 12 }}
```

Comment:
Does this line have to be indented two more spaces to match the others or does it not matter?